src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -516,6 +519,40 @@ def run(self, timeout):
       raise CorpusPruningException(
           'Corpus pruning failed to merge shared corpus\n' + repr(e))
 
+    return result.stats
+
+
+def record_cross_pollination_stats(
+    pruner_stats, pollinator_stats, project_qualified_name, sources, tags,
+    initial_corpus_size, minimized_corpus_size_units):
+  """Log stats about cross pollination in BigQuery."""
+  # BigQuery not available in local development.
```

Comment:
Nit: Could you give a slightly more detailed comment here? Without context it would look odd that we're checking if we're in a test here, so it's worth explaining that this is necessary because the untrusted runner is in a separate process and can't be easily mocked.